protos/feast/core/CoreService.proto

```diff
@@ -154,6 +155,9 @@ message ListStoresResponse {
 }
 
 message ApplyFeatureSetRequest {
+    // Feature set version and source will be ignored.
```

Comment:
Can you elaborate on the `source` being ignored?

Reply:
Sorry this documentation is just plainly incorrect. Source is not ignored if specified